<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modificarFornecedor</class>
 <widget class="QDialog" name="modificarFornecedor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Dados do Fornecedor</string>
  </property>
  <widget class="QLabel" name="labelModificarDados">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar Dados do Fornecedor:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkModificarDado">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>270</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelarModificarDados">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>270</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutModificarDadosFornecedores">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutNome">
      <item>
       <widget class="QLabel" name="labelNome">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNome"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutTelefone">
      <item>
       <widget class="QLabel" name="labelTelefone">
        <property name="text">
         <string>Telefone:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTelefone"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutEndereco">
      <item>
       <widget class="QLabel" name="labelEndereco">
        <property name="text">
         <string>Endere√ßo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEndereco"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
